### PR TITLE
Fixing the migration issues

### PR DESCRIPTION
**Pull request by bcdurak**
## Describe changes
I fixed the problems caused by the alembic migration.

## Pre-requisites
Please ensure you have done the following:
- [X] I have read the **CONTRIBUTING.md** document.
- [X] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/stacks-and-components/component-guide) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.
- [X] I have based my new branch on `develop` and the open PR is targeting `develop`. If your branch wasn't based on develop read [Contribution guide on rebasing branch to develop](https://github.com/zenml-io/zenml/blob/main/CONTRIBUTING.md#-pull-requests-rebase-your-branch-on-develop).

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: src/zenml/zen_stores/migrations/versions/37835ce041d2_optimizing_database.py [hunk 1]
```diff
@@ -202,13 +176,6 @@ def downgrade() -> None:
                 "parameters", sa.VARCHAR(length=16777215), nullable=False
             )
         )
-        batch_op.add_column(
```
Comment (fa9r) [inline src/zenml/zen_stores/migrations/versions/37835ce041d2_optimizing_database.py:205]: downgrade is not fully reflective of the upgrade changes

**Review by fa9r — CHANGES_REQUESTED** [review]
Changes look good to me, however before we merge I'd suggest we:
- combine the two new migrations into one
- either implement the downgrade correctly or just raise a `NotImplementedError`

**Comment by bcdurak** [comment]
@fa9r I implemented the suggested changes, merged two scripts into one, and put a `NonImplementedError` for the downgrade.

**Review by fa9r — COMMENTED** [review]
죽을래?

**Review by fa9r — APPROVED** [review]

File: src/zenml/zen_stores/migrations/versions/6f707b385dc1_fix_model_artifacts.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 # revision identifiers, used by Alembic.
 revision = "6f707b385dc1"
-down_revision = "0.45.0"
+down_revision = "7f603e583dd7"
 branch_labels = None
```
Comment (AlexejPenner) [inline src/zenml/zen_stores/migrations/versions/6f707b385dc1_fix_model_artifacts.py:14]: @bcdurak changing this in the upgrade tree seems very wrong to me, I think this is rewriting history in a way that might break things for people who are running on 0.45.0 now. Is this file somethign that was merged into develop after the 0.45.0 release?

Comment (AlexejPenner) [reply src/zenml/zen_stores/migrations/versions/6f707b385dc1_fix_model_artifacts.py:14]: ah went to investigate, all good with this.

**Pull request merged**